File: WebApi/Models/CardCommentModel.cs
using System;

namespace WebApi.Models
{
    public class CardCommentModel
    {
        public string Id { get; set; }
        public string Text { get; set; }
        public long CreateDateTimeInMiliseconds { get; set; }
        public bool NotUserComment { get; set; }
    }
}

File: AngularClient/src/app/modules/content/models/card-comment.model.ts
import { ICardCommentModel } from 'src/app/modules/content/@types/tasks';

export class CardCommentModel{
    id: string;
    text: string;
    createDateTime: Date;
    notUserComment: boolean;

    constructor(data: ICardCommentModel){
        this.id = data.id;
        this.text = data.text;
        this.notUserComment = data.notUserComment,
        this.createDateTime = new Date(data.createDateTime);
    }
}